hardhat.config.ts

```diff
@@ -496,6 +492,14 @@ const config: HardhatUserConfig = {
         ]
     },
 
+    paths: {
+        sources: "./contracts",
```

Comment:
Why are these necessary?

Reply:
removed the default paths - only left the deploy dir